pages/cart.html

```diff
@@ -0,0 +1,117 @@
+                        <i class="fas fa-shopping-cart fa-lg"></i>
+                        <div class="number badge-secondary text-white">2</div>
+                    </div>
+                </a>
+            </li>
+        </ul>
+    </nav>
+    <h3 class="text-center cart-heading">Cart(2)</h3>
+    <section class="cart-container flex-row">
+        <div class="cart-products flex-row">
+            <div class="horizontal-card flex-row">
+                <img class="image-responsive" src="../asset/headphones.jpg" loading="lazy" alt="product image">
+                <div class="card-details-container ">
+                    <h3 class="text-xl">Sony Headphones</h3>
+                    <p class="product-details text-base fw-500">Rs.41,990 <del class="fw-300 text-sm">Rs.49,990</del>
+                    </p>
+                    <p class="text-base fw-400">16% off </p>
+                    <p class="product-details  fw-500">Quantity:
+                        <button class="quantity-btn"><i class="fa-solid fa-circle-plus fa-lg"></i></button>
+                        <input value="1" name="name" type="text" class="input-area text-sm text-center quantity-input">
+                        <button class="quantity-btn"><i class="fa-solid fa-circle-minus fa-lg"></i></button>
+
+                    </p>
+                    <div class="cart-btn flex-column">
+                        <button class="btn btn-outlined-secondary ">Remove From Cart</button>
+                        <button class="btn btn-solid-primary  text-white">Move to Wishlist</button>
+                    </div>
+                </div>
+            </div>
+            <div class="horizontal-card flex-row">
+                <img class="image-responsive" src="../asset/ipad-air.png" loading="lazy" alt="product image">
+                <div class="card-details-container ">
+                    <h3 class="text-xl">iPad Air</h3>
+                    <p class="product-details text-base fw-500">Rs.41,990 <del class="fw-300 text-sm">Rs.49,990</del>
+                    </p>
+                    <p class="text-base fw-400">16% off </p>
+                    <p class="product-details  fw-500">Quantity:
+                        <button class="quantity-btn"><i class="fa-solid fa-circle-plus fa-lg"></i></button>
+                        <input value="1" name="name" type="text" class="input-area text-sm text-center quantity-input">
+                        <button class="quantity-btn"><i class="fa-solid fa-circle-minus fa-lg"></i></button>
+
+                    </p>
+                    <div class="cart-btn flex-column">
+                        <button class="btn btn-outlined-secondary ">Remove From Cart</button>
+                        <button class="btn btn-solid-primary  text-white">Move to Wishlist</button>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div class="price-container">
+            <h4 class="fw-500 ">Price Details</h4>
```

Comment:
what is fw-?

Reply:
its font-weight, i thought naming it something like font-weight-500 initially, but it was very lengthy,so i settled with fw-500.